python/src/nanoarrow/_lib.pyx

```diff
@@ -84,3 +85,329 @@ def as_numpy_array(arr):
     # TODO set base
 
     return result
+
+
+def version():
+    return ArrowNanoarrowVersion().decode("UTF-8")
+
+cdef class CSchemaHolder:
```

Comment:
Those "Holder" classes are only for when we "own" the pointer, and then to ensure t is properly released?  (to serve as the "base" object in that case, while if you create the CSchema/CArray class from another object that owns the pointer, this other object is the `_base`)